test/arena/psychic-terrain.test.ts

```diff
@@ -56,4 +56,30 @@ describe("Arena - Psychic Terrain", () => {
 
     expect(game.scene.arena.weather?.weatherType).toBe(WeatherType.RAIN);
   });
+
+  it("Quick claw boosted move is not blocked", async () => {
+    game.override.startingHeldItems([{ name: "QUICK_CLAW", count: 10 }]);
+    await game.classicMode.startBattle([SpeciesId.MAGIKARP]);
+
+    game.move.select(MoveId.PSYCHIC_TERRAIN);
+    await game.toNextTurn();
+
+    game.move.use(MoveId.POUND);
+    await game.toEndOfTurn();
+
+    expect(game.field.getEnemyPokemon().hp).toBeLessThan(game.field.getEnemyPokemon().getMaxHp());
+  });
```

Comment:
```suggestion
    await game.classicMode.startBattle([SpeciesId.FEEBAS]);
    
    game.move.use(MoveId.PSYCHIC_TERRAIN);
    await game.toNextTurn();

    expect(game).toHaveTerrain(TerrainType.PSYCHIC);
    const feebas = game.field.getPlayerPokemon();
    expect(allMoves[MoveId.POUND].getPriority(feebas)).toBe(0.2);

    game.move.use(MoveId.POUND);
    await game.toEndOfTurn();

    const shuckle = game.field.getEnemyPokemon();
    expect(shuckle).not.toHaveFullHp();
  });
```

don't forget to add some imports

Reply:
I could manually add the tag before the check and remove it after.